content.tex

```diff
@@ -63,7 +66,7 @@ \section{Rubin vNOC - TBD}
 
 However, if an alarm has not been acknowledged in the 20 minutes after Rubin's vNOC has reported it, they (A2: who?) will be instructed to execute an emergency escalation procedure, and call a list of staff in sequential mode until someone answers the phone.
 
-The Rubin vNOC could be a separate instance from the LHN vNOC and despite Rubin's vNOC focus is the infrastructure in Chile, some LHN data points will also be monitored.
+The Rubin vNOC could be a separate instance from the Long Haul Network (LHN) vNOC and despite Rubin's vNOC focus is the infrastructure in Chile, some LHN data points will also be monitored.
 
 \textit{
 (A2 Note:  Not sure if this is generic and all applicable across all teams - please advise).}
```

Comment:
I think IT specific... which means the Alerting section should probably be a subsection.

Reply:
Alerts to On-Call Support and In-Person Emergency Support staff and can be activated by the people present at the summit during on-sky observing.  It is anticipated this will be done using a call list.

Additionally, alerts will also be triggered by the monitoring stack and routed to the incident management tool using the commodity internet.  These automated alerts will more likely be DevOps related.

In DevOps the incident management tool will control the shift rotations, escalations, and will alert according to the severity of the fault.